client/signup/step-wrapper/index.jsx

```diff
@@ -1,20 +1,49 @@
+import { isEnabled } from '@automattic/calypso-config';
+import { Button } from '@automattic/components';
+import { useGetZendeskFieldsByFlow } from '@automattic/help-center/src/hooks';
+import { HELP_CENTER_STORE } from '@automattic/help-center/src/stores';
 import { ActionButtons } from '@automattic/onboarding';
+import { useDispatch, useSelect as useDataStoreSelect } from '@wordpress/data';
 import clsx from 'clsx';
 import { localize } from 'i18n-calypso';
 import PropTypes from 'prop-types';
 import { Component } from 'react';
 import { connect } from 'react-redux';
 import FormattedHeader from 'calypso/components/formatted-header';
-import { usePresalesChat } from 'calypso/lib/presales-chat';
 import flows from 'calypso/signup/config/flows';
 import NavigationLink from 'calypso/signup/navigation-link';
 import { isUserLoggedIn } from 'calypso/state/current-user/selectors';
 import getCurrentQueryArguments from 'calypso/state/selectors/get-current-query-arguments';
 import './style.scss';
 
-function PresalesChat() {
-	usePresalesChat( 'wpcom' );
-	return null;
+function HelpCenterButton( { helpCenterButtonText, hasPremiumSupport, flowName } ) {
+	const { setShowHelpCenter, setNavigateToRoute } = useDispatch( HELP_CENTER_STORE );
+	const isShowingHelpCenter = useDataStoreSelect(
+		( select ) => select( HELP_CENTER_STORE ).isHelpCenterShown(),
+		[]
+	);
```

Comment:
It's slightly worrisome that we had the Help Center loaded in signup this whole time for little to no reason 🐌